plugins/inx/server_blocks.go

```diff
@@ -55,25 +72,15 @@ func INXNewBlockMetadata(blockID iotago.BlockID, metadata *storage.BlockMetadata
 			return nil, errors.WithMessage(echo.ErrInternalServerError, err.Error())
 		}
 
-		var shouldPromote bool
-		var shouldReattach bool
-
 		switch tipScore {
 		case tangle.TipScoreNotFound:
 			return nil, errors.WithMessage(echo.ErrInternalServerError, "tip score could not be calculated")
 		case tangle.TipScoreOCRIThresholdReached, tangle.TipScoreYCRIThresholdReached:
-			shouldPromote = true
-			shouldReattach = false
+			m.ShouldPromote = true
```

Comment:
can we leave comments here which say that promote and reattach are false?
I liked to keep the setting of the explicit value so it is easier to read.
but a comment should also be fine :)
```suggestion
			m.ShouldPromote = true
			// reattach is false
```